docs/appendices/release-notes/unreleased.rst

```diff
@@ -112,4 +112,5 @@ Changes
 Fixes
 =====
 
-None
+- Fixed a parsing issue for HBA SSL configuration.
```

Comment:
I think this could be a bit more specific - so that a end-user would understand what didn't work / what was fixed.